src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java

```diff
@@ -92,6 +92,11 @@ public boolean isLightweight() {
         this.lightweight = lightweight;
     }
 
+    @SuppressFBWarnings(
+            value = {"NP_LOAD_OF_KNOWN_NULL_VALUE", "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE",
+                    "RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE", "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"},
+            justification = "false positives for try-resource in java 11"
```

Comment:
This looks familiar; do you have a link to the SpotBugs issue offhand?

Reply:
https://github.com/spotbugs/spotbugs/issues/756